presto-main/src/main/java/io/prestosql/server/protocol/Query.java

```diff
@@ -618,7 +618,7 @@ private static StatementStats toStatementStats(QueryInfo queryInfo)
                 .setElapsedTimeMillis(queryStats.getElapsedTime().toMillis())
                 .setProcessedRows(queryStats.getRawInputPositions())
                 .setProcessedBytes(queryStats.getRawInputDataSize().toBytes())
-                .setPeakMemoryBytes(queryStats.getPeakUserMemoryReservation().toBytes())
+                .setPeakMemoryBytes(queryStats.getPeakUserMemoryReservation().toBytes() + queryStats.getPeakNonRevocableMemoryReservation().toBytes())
```

Comment:
`getPeakNonRevocableMemoryReservation()` was added in 337: https://github.com/trinodb/trino/commit/11b7d194bb0171840f383980afb50701f9605ba5

Reply:
I've changed it to `getPeakTotalMemoryReservation` here. As we don't use evocableMemory (spill), the values will be the same. In 350, we can use `getPeakNonRevocableMemoryReservation`.